pkg/app/piped/trigger/trigger.go

```diff
@@ -232,12 +232,17 @@ func (t *Trigger) checkRepoCandidates(ctx context.Context, repoID string, cs []c
 			)
 
 			// Set ApplicationSyncState to INVALID_CONFIG when LoadApplication fails.
+			shortReasonLimit := 100
+			if len(err.Error()) < shortReasonLimit {
+				shortReasonLimit = len(err.Error())
+			}
 			req := &pipedservice.ReportApplicationSyncStateRequest{
 				ApplicationId: app.Id,
 				State: &model.ApplicationSyncState{
-					Status:    model.ApplicationSyncStatus_INVALID_CONFIG,
-					Reason:    err.Error(),
-					Timestamp: time.Now().Unix(),
+					Status:      model.ApplicationSyncStatus_INVALID_CONFIG,
+					ShortReason: err.Error()[:shortReasonLimit],
```

Comment:
```suggestion
					ShortReason: "failed to load application configuration",
```

Reply:
I think this statement is the same meaning as the status INVALID_CONFIG and is not necessary. Instead, if we display part of the actual error (in most cases, the full text, since error messages are not long) in shortReason, the user can easily see the error. What do you think?

Reply:
I think this is a UI problem, since users can see the "actual" error message in detail page; showing it here is redundant is most case. The same idea with `Some thing went wrong` text message on other services; users are supposed to check the error message (along with other details information) in detail page rather than showing everything in the listing page. 

And besides, INVALID_CONFIG is the status and it's displayed by icon, not everyone have sense of what it is by seeing the icon, but text like this would help. WDYT? 🤔

Reply:
And beisdes, it's safer for the UI to be not broken (by long text) without adding unneeded if condition to cut the text if it's longer than limit or thing.

Reply:
I see, I understand. Thanks for the good review.